<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideViewDialog</class>
 <widget class="QWidget" name="SlideViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="AspectContainer" name="aspectWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="SizeGraphicsView" name="slideView">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorViewCenter</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="title">
      <string>View settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_14">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Exported size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="slideXEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>The dimensions used when saving.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="slideYEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>The dimensions used when saving.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QWidget" name="settingsFormWidget" native="true">
        <layout class="QFormLayout" name="formLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>Lines of comment text:</string>
           </property>
           <property name="buddy">
            <cstring>slideLinesSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="slideLinesSpinBox">
           <property name="toolTip">
            <string>This controls the font size for the comments display</string>
           </property>
           <property name="minimum">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Comment margin:</string>
           </property>
           <property name="buddy">
            <cstring>slideMarginSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="slideMarginSpinBox">
           <property name="maximum">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="slideBoldCheckBox">
        <property name="toolTip">
         <string>Determines how the first paragraph of the comments should be displayed.</string>
        </property>
        <property name="text">
         <string>Bold header</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="slideWBCheckBox">
        <property name="toolTip">
         <string>Choose the color scheme for the comments</string>
        </property>
        <property name="text">
         <string>White on black text</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="slideCoordsCheckBox">
        <property name="toolTip">
         <string>Show board coordinates</string>
        </property>
        <property name="text">
         <string>Coordinates</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="slideItalicCheckBox">
        <property name="toolTip">
         <string>Determines how the first paragraph of the comments should be displayed.</string>
        </property>
        <property name="text">
         <string>Italic header</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Automatic file naming (numbered sequence)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSpinBox" name="fileNrSpinBox">
          <property name="toolTip">
           <string>Counts the number of the slide, incremented on every save.
This is substituted into the filename template when choosing the Save button.</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="overwriteCheckBox">
          <property name="toolTip">
           <string>If checked, the Save button overwrites files without asking</string>
          </property>
          <property name="text">
           <string>&amp;Force overwrite</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="fileTemplateEdit">
        <property name="toolTip">
         <string>File name template
This should be a filename, with &quot;%n&quot; somewhere to indicate where the current number should be substituted.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="fileOpenButton">
        <property name="toolTip">
         <string>Choose a file name template
You will need to add '%n&quot; somewhere in the template to indicate the position where the current number should be substituted.</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="q4go.qrc">
          <normaloff>:/ClientWindowGui/images/clientwindow/fileopen.png</normaloff>:/ClientWindowGui/images/clientwindow/fileopen.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Current number:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Save and increment</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2">
          <widget class="QPushButton" name="saveAllMainButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>All positions on main branch</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="saveComButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>All commented positions</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="saveComMainButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>Commented positions on main branch</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="saveAllButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>All positions</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="saveButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>Current position</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Filename pattern (%n for number):</string>
        </property>
        <property name="buddy">
         <cstring>fileTemplateEdit</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="toClipButton">
       <property name="toolTip">
        <string>Export the current slide to the clipboard</string>
       </property>
       <property name="text">
        <string>To &amp;clipboard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveAsButton">
       <property name="toolTip">
        <string>Choose a file name and save the current slide, without using the template.</string>
       </property>
       <property name="text">
        <string>Save &amp;as...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AspectContainer</class>
   <extends>QWidget</extends>
   <header>sizegraphicsview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SizeGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>sizegraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="q4go.qrc"/>
 </resources>
 <connections/>
</ui>
